CHANGELOG.md

```diff
@@ -86,7 +86,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Reduced duplicate code
 - Updated CIS playbook to Orchestrator architecture
 - Single Orchestrator deployment to enable multi-standard remediation with a single click
-- Custom Actions now consolidated to one: "Remediate with SHARR"
+- Custom Actions now consolidated to one: "Remediate with ASR"
```

Comment:
No need to change the old changelog entry, technically it's still correct

Reply:
Whoops, didn't realize the file name when I was bulk changing